tests/mod_glossary/comment_created/comment_created_test.php

```diff
@@ -0,0 +1,70 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace logstore_xapi\mod_glossary;
+
+defined('MOODLE_INTERNAL') || die();
+
+global $CFG;
+
+require_once($CFG->dirroot . '/admin/tool/log/store/xapi/tests/xapi_test_case.php');
+
+/**
+ * Unit test for mod_glossary comment created event.
+ * 
+ * @package   logstore_xapi
+ * @copyright Cliff Casey <cliff@yetanalytics.com>
+ * @license   https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class comment_created_test extends \logstore_xapi\xapi_test_case {
+
+    /**
+     * Retrieve the directory of the unit test.
+     *
+     * @return string
+     */
+    protected function get_test_dir() {
+        return __DIR__;
+    }
+
+    /**
+     * Retrieve the plugin type being tested.
+     *
+     * @return string
+     */
+    protected function get_plugin_type() {
+        return "mod";
+    }
+
+    /**
+     * Retrieve the plugin name being tested.
+     *
+     * @return string
+     */
+    protected function get_plugin_name() {
+        return "glossary";
+    }
+
+    /**
+     * Appease auto-detecting of test cases. xapi_test_case has default test cases.
+     *
+     * @covers ::discussion_viewed
```

Comment:
This annotation is wrong (in all three tests). I only recently noticed it myself and have probably also pasta'd the wrong thing in there. No idea what it does.

Reply:
done